ui/src/JobDetails.js

```diff
@@ -255,7 +258,12 @@ class JobDetails extends Component {
             {jobDetailsFragment}
             {buttonFragment}
             {this.props.job &&
-              this.props.authorisedActivities.includes("LOAD_SAMPLE") &&
+              (this.props.job.jobType === "SAMPLE" &&
+                this.props.authorisedActivities.includes("LOAD_SAMPLE")) ||
+                (this.props.job.jobType === "BULK_REFUSAL" &&
+                  this.props.authorisedActivities.includes(
+                    "LOAD_BULK_REFUSAL"
+                  ))) &&
```

Comment:
I would drive this via a prop

for example....

```js
this.props.authorisedActivities.includes(this.props.loadPermission)
```

Then just supply that prop in the component above, for example:
```
  <JobDetails loadPermission="LOAD_SAMPLE">
```